Client/serverConnect.cs

```diff
@@ -8,6 +8,41 @@
 using Newtonsoft.Json.Serialization;
 using System.Diagnostics;
 
+namespace SNUBclientFinalProject
+{
+    // Watched a youtube tutorial 0 warnings now
+    // had to change a bit to fix one of the errors
+    // because said to use HTTP instead of webclient warning
+
+
+    internal class serverConnect
+    {
+        HttpClient client = new HttpClient();
+
+        static async Task Main(string[] args)
+        {
+            serverConnect serverConnect = new serverConnect();
+            await serverConnect.GetTodoItems();
+        }
+        private async Task GetTodoItems()
+        {
+            string response = await client.GetStringAsync(
+                "https://snub.kitsunehosting.net");
+        }
+
+    }
+}
+/* OLD CODE
+ * using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System.Diagnostics;
+
```

Comment:
Was this marked "old code?" you should not leave old code hanging. Git remembers every line ever added so you can reference the past directly.